ui/css/treeherder-fuzzyfinder.css

```diff
@@ -0,0 +1,8 @@
+/*
+ * Fuzzy runnable job finder
+ */
+
+.fuzzy-modal .modal-body #addJobsGroup option.selected::before {
```

Comment:
Now that #4489 has landed, this will need a rebase and then a `yarn format` so the CSS style in this newly added file is updated :-)